src/main/java/gift/dto/LoginRequest.java

```diff
@@ -0,0 +1,33 @@
+package gift.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+
+public class LoginRequest {
+
+    @NotBlank(message = "이메일는 필수 입력 값입니다.")
+    @Pattern(regexp = "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}$", message = "유효한 이메일 형식이 아닙니다.")
```

Comment:
이메일 유효성 검사는 정의된 어노테이션을 활용할 수 있어요! 

```suggestion
    @Email(message = "유효한 이메일 형식이 아닙니다.")
```